weave

```diff
@@ -813,6 +813,16 @@ dns_args() {
     fi
 }
 
+rewrite_hosts() {
+    path=$(docker inspect -f '{{.HostsPath}}' $CONTAINER)
+    name=$(docker inspect -f '{{.Config.Hostname}}' $CONTAINER)
+    containers_path="$(dirname $(dirname "$path"))"
+    docker run --rm \
+        -v "$containers_path":"$containers_path" \
+        --entrypoint=/bin/sh \
+        $EXEC_IMAGE -c "/usr/bin/weavehosts $path $name $ALL_CIDRS"
```

Comment:
I see.

But in that case, couldn't we mount just the specific file instead of the dir?

Also, why is the entrypoint `/bin/sh` instead of `/usr/bin/weavehosts`? Seems like an unnecessary indirection.